integration-tests/src/test/java/oracle/weblogic/kubernetes/utils/ItMiiSampleHelper.java

```diff
@@ -49,27 +49,24 @@
 @IntegrationTest
 public class ItMiiSampleHelper {
 
-  private static final String MII_SAMPLES_WORK_DIR = RESULTS_ROOT
-      + "/model-in-image-sample-work-dir";
-  private static final String MII_SAMPLES_SCRIPT =
-      "../operator/integration-tests/model-in-image/run-test.sh";
-
-  private static final String CURRENT_DATE_TIME = getDateAndTimeStamp();
-  private static final String MII_SAMPLE_WLS_IMAGE_NAME_V1 = DOMAIN_IMAGES_REPO + "mii-" + CURRENT_DATE_TIME + "-wlsv1";
-  private static final String MII_SAMPLE_WLS_IMAGE_NAME_V2 = DOMAIN_IMAGES_REPO + "mii-" + CURRENT_DATE_TIME + "-wlsv2";
-  private static final String MII_SAMPLE_JRF_IMAGE_NAME_V1 = DOMAIN_IMAGES_REPO + "mii-" + CURRENT_DATE_TIME + "-jrfv1";
-  private static final String MII_SAMPLE_JRF_IMAGE_NAME_V2 = DOMAIN_IMAGES_REPO + "mii-" + CURRENT_DATE_TIME + "-jrfv2";
-  private static final String SUCCESS_SEARCH_STRING = "Finished without errors";
-
-  private static String opNamespace = null;
-  private static String domainNamespace = null;
-  private static String traefikNamespace = null;
-  private static String dbNamespace = null;
-  private static Map<String, String> envMap = null;
-  private static LoggingFacade logger = null;
-  private static boolean previousTestSuccessful = true;
-  private static DomainType domainType = null;
-  private static ImageType imageType = null;
+  private final String miiSampleScript = "../operator/integration-tests/model-in-image/run-test.sh";
+
+  private final String currentDateTime = getDateAndTimeStamp();
+  private final String miiSampleWlsImageNameV1 = DOMAIN_IMAGES_REPO + "mii-" + currentDateTime + "-wlsv1";
+  private final String miiSampleWlsImageNameV2 = DOMAIN_IMAGES_REPO + "mii-" + currentDateTime + "-wlsv2";
+  private final String miiSampleJrfImageNameV1 = DOMAIN_IMAGES_REPO + "mii-" + currentDateTime + "-jrfv1";
+  private final String miiSampleJrfImageNameV2 = DOMAIN_IMAGES_REPO + "mii-" + currentDateTime + "-jrfv2";
```

Comment:
Minor comment: If two parallel runs occur within the same second, then the image names still can overlap. If you want to fix this, then I think this can be addressed by decorating the image name with the uniquely generated domainNamespace.  For example, a new helper function `modelImageName(String suffix)` could return `DOMAIN_IMAGES_REPO + "mii-" + currentDateTime + "-" + domainNamespace + suffix`, and `modelImageName(-wlsv1")` could then be used everywhere instead of `miiSampleWlsImageNameV1`.

Reply:
good idea. I will create a new task to fix this later since we have to fix the failure on nightly ASAP and I have ran jenkins jobs several days.

Reply:
JIRA https://jira.oraclecorp.com/jira/browse/OWLS-92044 is created